Userland/Libraries/LibGfx/ImageFormats/PNGShared.h

```diff
@@ -56,12 +57,20 @@ ALWAYS_INLINE u8 paeth_predictor(u8 a, u8 b, u8 c)
 
 ALWAYS_INLINE AK::SIMD::u8x4 paeth_predictor(AK::SIMD::u8x4 a, AK::SIMD::u8x4 b, AK::SIMD::u8x4 c)
 {
-    return AK::SIMD::u8x4 {
-        paeth_predictor(a[0], b[0], c[0]),
-        paeth_predictor(a[1], b[1], c[1]),
-        paeth_predictor(a[2], b[2], c[2]),
-        paeth_predictor(a[3], b[3], c[3]),
-    };
+    AK::SIMD::i16x4 a16 = AK::SIMD::simd_cast<AK::SIMD::i16x4>(a);
```

Comment:
Maybe some `auto`s would make that a little bit lighter.

Reply:
or some `using` directives

Reply:
Done.